features/networking/pod.feature

```diff
@@ -480,3 +480,39 @@ Feature: Pod related networking scenarios
     | ip route |
   Then the output should contain:
     | <%= cb.pod_cidr %> dev <%= cb.tunnel_inf_name %> |
+    
+  # @author anusaxen@redhat.com
+  # @case_id OCP-26373
+  @admin
+  @destructive  
+  Scenario: Pod readiness check for OVN		
+  Given the env is using "OVNKubernetes" networkType
+  And OVN is functional on the cluster
+  Given I switch to cluster admin pseudo user
+  And I use the "openshift-ovn-kubernetes" project
+  And a pod is present with labels:
+    | l | app=ovnkube-node |
+  #Removing CNI config file from container to check readiness probe functionality
+  When I run the :exec client command with:
+    | pod              | <%= pod.name %>                       |
+    | c                | ovnkube-node                          |
+    | oc_opts_end      |                                       |
+    | exec_command     | rm                                    |
+    | exec_command_arg | /etc/cni/net.d/10-ovn-kubernetes.conf |
+  Then the step should succeed
+  #Deleting ovnkube-pod will force CNO to rewrite the conf file and bring cluster back to normal after scenario
+  And I register clean-up steps:
+  """
+  admin ensure "<%= pod.name %>" pod is deleted from the "openshift-ovn-kubernetes" project
+  And OVN is functional on the cluster
+  """
+  #Now make sure readiness probe checking above file will cause one of the two ovnkube-node containers to go down and container ready status change to false
+  Given I wait up to 30 seconds for the steps to pass:
+  """
+  When I run the :get admin command with:
+    | resource      | pod                                             |
+    | resource_name | <%= pod.name %>                                 |
+    | o             | jsonpath='{.status.containerStatuses[1].ready}' |
```

Comment:
we might be able to use a file in jsonpath and not rely on ordering of containers.

`'{.status.containerStatuses[?(@.name=="ovnkube-node")].ready}'`

Reply:
Thats a great suggestion, Ross. Was never able to crack this json code..heh. Fixed!